src/bindings/python/src/openvino/runtime/opset10/ops.py

```diff
@@ -10,6 +10,7 @@
 from openvino.runtime.opset_utils import _get_node_factory
 from openvino.runtime.utils.decorators import nameable_op
 from openvino.runtime.utils.types import (
+    NumericType,
```

Comment:
Should it be here? I do not see a use of it in changes.

Reply:
Removed